### ISSUE TITLE
Getting plugin information can fail when resolution is happening concurrently

### ISSUE THREAD

**Issue by nmiyake** [palantir/godel#535]
## What happened?
When running multiple copies of godel concurrently with an empty GODEL_HOME, some of the operations failed with errors like the following:

```
Error: failed to resolve 2 plugin(s):
    failed to get plugin info for plugin com.palantir.godel-format-plugin:format-plugin:1.6.0: command [/local/.godel/plugins/com.palantir.godel-format-plugin-format-plugin-1.6.0 _godelPluginInfo] failed.
Error:
fork/exec /local/.godel/plugins/com.palantir.godel-format-plugin-format-plugin-1.6.0: text file busy
Output:
```

This is related to #533 

## What did you want to happen?
Resolution should succeed even when run concurrently

**Comment by nmiyake** [comment]
Re-opening because issue can still be observed even after #536

**Comment by nmiyake** [comment]
Fixed by #537
